src/obfl-specification.html

```diff
@@ -1077,7 +1078,8 @@ <h4 id="L1023" class="include">block</h4>
   <dt>Usage</dt>
     <dd>before, after, sequence, dynamic-sequence, on-collection-start,
       on-page-start, on-page-end, on-collection-end, on-toc-start,
-      on-volume-start, on-volume-end, on-toc-end, item, td</dd>
+      on-volume-start, on-volume-end, on-toc-end, item, td, block,
+      toc-block</dd>
```

Comment:
The additional of `block` is correct. But it should not be allowed inside `toc-block`.